kafka-ballerina/records.bal

```diff
@@ -58,6 +58,17 @@ public type SecureSocket record {|
    string provider?;
 |};
 
+# Configurations related to the CertKey record required in `kafka:SecureSocket` record.
```

Comment:
Shall we update the doc as in https://docs.central.ballerina.io/ballerina/http/1.1.0-beta.1/records/CertKey

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/240/commits/0e175487046422ca076a49b0608096d6774fedfa